modules/multi-arch-installing-using-web-console.adoc

```diff
@@ -0,0 +1,49 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-installing-using-web-console_{context}"]
+
+= Installing Multiarch Tuning Operator by using the web console
+
+You can install the Multiarch Tuning Operator by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Click *Operators -> OperatorHub*.
+. Enter *Multiarch Tuning Operator* in the search field.
+. Click *Multiarch Tuning Operator*.
+. Select the *Multiarch Tuning Operator* version from the *Version* list.
+. Click *Install*
+. Set the following options on the *Operator Installation* page:
+.. *Update Channel* as *tech-preview*.
+.. *Installation Mode* as *All namespaces on the cluster*.
+.. *Installed Namespace* as *Operator recommended Namespace* or *Select a Namespace*.
++
+The default Operator namespace is `openshift-multiarch-tuning-operator`. If the `openshift-multiarch-tuning-operator` namespace does not exist, it is created during the operator installation.
++
+If you select *Select a namespace*, you must select a namespace for the Operator from the *Select Project* list.
```

Comment:
@lwan-wanglin now that https://issues.redhat.com/browse/MULTIARCH-4598 is closed, can you do some smoke testing for the case when the operator runs on a different namespace than the default?

Reply:
ok

Reply:
the operator running on a different namespace than default works well.